samples/net/coap_client/src/coap-client.c

```diff
@@ -264,6 +305,45 @@ static void event_iface_up(struct net_mgmt_event_callback *cb,
 	/* Increase packet ref count to avoid being unref after sendto() */
 	coap_pending_cycle(pending);
 
+
+	switch (code) {
```

Comment:
if we consolidate two payloads variable into one then we can replace the following as
`
if (code == COAP_METHOD_PUT || code == CODE_METHOD_POST) {
		r = coap_packet_append_payload_marker(&request);
		if (r) {
			net_pkt_unref(pkt);
			printk("Unable to append payload marker\n");
		}

		r = coap_packet_append_payload(&request, (u8_t *)put_payload,
				                                       strlen(put_payload));
		if (r < 0) {
			net_pkt_unref(pkt);
			printk("Not able to append payload\n");
		}
}
`

Reply:
I want to use different payload message for both methods. So that I have separated out.